app/views/layouts/hera/_project_nav.html.erb

```diff
@@ -0,0 +1,71 @@
+<nav class="navbar navbar-expand-lg project-nav">
+  <div class="container-fluid justify-content-end p-lg-0">
+    <div class="collapse navbar-collapse dual-nav" id="project_navbar" data-behavior="navbar-collapse">
```

Comment:
Ditto with the id. it's in a different format than the one above, if we need them let's make the case consistent